### ISSUE TITLE
re-FAQ-ification?

### ISSUE THREAD

**Issue by coreyjkelly** [StatsBen/ubc-aviary#5]
I _do_ love how much character our FAQ has, but I am beginning to wonder if it could be more effective. The info email and the Facebook page still get the same questions over and over, despite the answers being in the FAQ section. 

A few thoughts:
- I think the FAQ at the bottom of the page makes the most sense, but is there a way we can direct more users there? A link in the nav bar helps a bit, but something more obvious at the top of the page might be better still. What can we do without being super obnoxious to the majority of users who just want to scroll to the hours every time they visit.
- Could the FAQ be more tersely written, while still being fun? I personally like to try to find the answers to questions on my own, but the majority of users are choosing the second option here, and I can't blame them:
  1. Scroll to the bottom of the page, skim through, read the bit about bouldering, still be unclear on whether there is _dedicated_ bouldering terrain at The Aviary. (~minutes)
  2. Click the email link, type "is boldring at avary?", send. (~seconds)
- People are invariably confused about how to get started climbing. These parts in particular could be more prominently displayed and clearly written.

To be clear, not all of this is web-design-related. I'm going to bring up the FAQ at the next Exec meeting and see if anyone is interested in doing a re-write. I'm mostly curious about design decisions we can make, independent of the FAQ contents, that will help these issues.

**Comment by StatsBen** [comment]
I completely agree that if we want to reduce emails to info@ with easily answered questions, this FAQ section is terrible! 

I'm not attached to it at all, and I think it should be upgraded... I think I only wrote it like this in the first place because I thought it'd be so annoying that a current exec member would be motivated to write some real ones.

I think there's two options here:
- _easiest_: Find a willing, current exec member with up-to-date Aviary policy knowledge to write it.
- _harder_: I could try to export the 'info@' mailbox, compute some statistics about the most frequent questions, compile them, and then have a current exec member write answers.

An option we shouldn't consider:
- yet another task getting lumped on Corey :'(

Let me know how the pitch goes at the next exec meeting and I'll be ready to support! Talk to you soon, Corey.
